python/lsst/verify/bin/inspectjob.py

```diff
@@ -19,16 +19,17 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-"""Print the contents of a persisted Job object.
+"""Print the contents of an lsst.verify.Job file.
 
-This script takes as arguments one or more Job .json files, and prints the
-top-level metadata and a summary of any measurements. It does not print
-metrics that don't have measurements (there are far too many) or
-specifications (which are not helpful for testing measurement code).
+This script takes as arguments one or more lsst.verify JSON files, and prints
+the top-level metadata and a summary of any measurements.
+
+It does not print metrics that don't have measurements or specifications.
```

Comment:
From the other docs, it sounded like this doesn't care about specifications.  Is that not true?

Reply:
Aha, I misread the original documentation. I should be that it just doesn't print specifications at all.